CONTRIBUTING.md

```
@@ -0,0 +1,168 @@
+* Manage Something
+  * [Provide Support on Issues](#provide-support-on-issues)
+  * [Label Issues](#label-issues)
+  * [Clean Up Issues and PRs](#clean-up-issues-and-prs)
+  * [Merge Pull Requests](#merge-pull-requests)
+
+
+## Request a Feature
+
+If the project doesn't do something you need or want it to do:
+
+* Open an Issue at https://github.com/canopas/group-track-android/issues
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)